openaerostruct/utils/testing.py

```diff
@@ -217,6 +217,281 @@ def get_ground_effect_surfaces():
     return surfaces
 
 
+def get_three_section_surface_sym():
```

Comment:
Could you clean-up these util functions and reduce code duplication?
For example, the only differences (I believe) between `get_three_section_surface_sym` and `get_three_section_surface_asym` are the `symmetry` and `root_section` entries. If so, you don't need to define `surface_dict` from scratch twice. You can first define it for let's say symmtry=True, and for the other util function, you can get that surface and overwrite `symmtry=False`.
Same for `get_two_section_surface_*`

Reply:
Done